examples/plugins/writing-plugins.rst

```diff
@@ -0,0 +1,131 @@
+Writing Your Own MLflow Plugin(s)
+---------------------------------
+
+Introduction
+~~~~~~~~~~~~
+
+MLflow Python API plugins provide a powerful mechanism for customizing the behavior of the MLflow
+Python client, allowing you to define custom behaviors for logging metrics, params, and artifacts,
+set special context tags at run creation, and override model registry methods for registering
+models etc.
+
+Defining a Plugin
+~~~~~~~~~~~~~~~~~
+MLflow plugins are defined as standalone Python packages which can then be distributed for
+installation via PyPI or conda. See https://github.com/mlflow/mlflow/tree/master/tests/resources/mlflow-test-plugin for an
```

Comment:
MLflow plugins are defined as standalone Python packages -> You define an MLflow plugin as a standalone Python package